src/sentry/incidents/subscription_processor.py

```diff
@@ -550,6 +551,17 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                 ):
                     # NOTE: There should only be one anomaly in the list
                     for potential_anomaly in potential_anomalies:
+                        # check to see if we have enough data for the dynamic alert rule now
+                        if self.alert_rule.status == AlertRuleStatus.NOT_ENOUGH_DATA.value:
+                            if self.has_enough_data(potential_anomaly):
+                                # NOTE: this means "enabled," and it's the default alert rule status.
+                                # TODO: change these status labels to be less confusing
+                                self.alert_rule.status = AlertRuleStatus.PENDING.value
+                                self.alert_rule.save()
+                            else:
+                                # we don't need to check if the alert should fire if the alert can't fire yet
+                                continue
```

Comment:
Do we need to continue iterating through this - like would there be a situation in which the next `potential_anomaly` would change the result of `self.has_enough_data()`? If not, we can just break here (which I realize is also a little silly since we expect one anomaly, but 🤷‍♀️

Reply:
I don't *think* so, but I also don't know, which is why I did continue instead of break. If only one anomaly is returned, like how it's spec'd in Seer, then they accomplish the same thing. I think continue is just a little safer.

Reply:
🤔 

Would Seer ever actually return any potential anomalies if we don't have enough data?
This check may actually be unnecessary? (but safe to have anyways)